interfaces/HO-Portal/src/app/program/metrics-states/metrics-states.component.html

```diff
@@ -1,10 +1,19 @@
 
 <table class="metrics-states--table">
   <caption>
-    <app-refresh-data
-      [lastUpdated]="lastUpdated"
-      (refresh)="update()"
-    ></app-refresh-data>
+    <ion-row>
+      <ion-col>
+        <ion-button float-left id="btn-export-csv" (click)="exportCSV()" size="small" >
```

Comment:
```suggestion
        <ion-button
          (click)="exportCSV()"
          size="small"
          class="ion-float-left"
        >
```

We try to use the 'regular' CSS class-names, instead of the (deprecated) attributes for alignment/layout.
Also, if an ID isn't necessary, we can leave it out.

And the HTML isn't being formatted automatically, but the 'rule of thumb' is "when there are more then 2 attributes, align them as 1-indent for every attribute on its own line".

(But there are places where this rule isn't applied. ;) )